templates/single-campaigns.twig

```diff
@@ -0,0 +1,93 @@
+{% extends "base.twig" %}
+
+{% block content %}
+
+	<div class="clearfix"></div>
+	<div class="skewed-overlay"></div>
+
+	<article id="post-{{ post.ID }}">
+
+		<header class="page-header">
+			<div class="page-header-background">
+				{#<img src="{{ background_image }}" class="page-header-image" alt="Image alt data"/>#}
+			</div>
+			<div class="container">
+				<div class="top-page-tags">
+					{% if ( page_type ) %}
+						<a class="tag-item tag-item--main page-type" href="{{ fn('get_term_link', page_term_id) }}">{{ page_type|e('wp_kses_post')|raw }}</a>
+					{% endif %}
+
+					{% if ( post.issues_nav_data ) %}
+						<div class="tag-wrap issues">
+							{% for issue in post.issues_nav_data %}
+								<a class="tag-item tag-item--main" href="{{ issue.link|default('#') }}">{{ issue.name|e('wp_kses_post')|raw }}</a>
+							{% endfor %}
+						</div>
+					{% endif %}
+
+					{% if (post.tags) %}
+						<div class="tag-wrap tags">
+							{% for tag in post.tags %}
+								<a class="tag-item tag" href="{{ tag.link }}">#{{ tag.name|e('wp_kses_post')|raw }}</a>
+							{% endfor %}
+						</div>
+					{% endif %}
+				</div>
+				<h1 class="page-header-title">{{ post.title|raw }}</h1>
+				<div class="row">
+					<div class="col-md-6">
+						<div class="single-post-meta">
+							<address class="single-post-author">
+								{{ __( 'by', 'planet4-master-theme' ) }}
```

Comment:
Constructing the 'by' line like this is not really translatable. For example if a particular language puts the equivalent of 'by' after the author name. It may be better to do a sprintf in the php and add it to the context.